pkg/middleware/validation.go

```diff
@@ -32,7 +31,11 @@ func NewValidationMiddleware(entry log.Entry, validatorHook string) buffalo.Midd
 			if version != "" {
 				valid, err := validate(validatorHook, mod, version)
 				if err != nil {
-					entry.SystemErr(err)
+					// Not sure if we can create a `SystemErr` function instead of this?
```

Comment:
yes, we should figure this out because I have Athens deployed in GCP and I won't be able to correctly observe things :)

Reply:
would you be cool with [this](https://github.com/gomods/athens/pull/645/files#r217569404) if it had basically the same code as `lggr.SystemErr` in it?